packages/core/aqua-to-js/src/utils.ts

```diff
@@ -27,24 +27,26 @@ import {
   SimpleTypes,
   UnlabeledProductType,
 } from "@fluencelabs/interfaces";
+import { z } from "zod";
 
-export interface PackageJson {
-  name: string;
-  version: string;
-  devDependencies: {
-    ["@fluencelabs/aqua-api"]: string;
-  };
-}
+const packageJsonSchema = z.object({
+  name: z.string(),
+  version: z.string(),
+  devDependencies: z.object({
+    // This version used in header file
```

Comment:
```suggestion
    // This version used in header file
```
I persononally don't see a lot of value in having this comment, since anybody looking through the code base can pretty quickly find where it used and why if curious, but if we include comment - probably something like this would be a bit more informative to the reader
```suggestion
    // @fluencelabs/aqua-api version is included as part of the comment at the top of each js and ts file
```